### PR TITLE
Simplifies the pull request template

### PR DESCRIPTION
**Pull request by PowerfulBacon**
Simplifies the PR template, removing a lot of blank space in the details section and simplifying an unnecessarilly long comment in the testing photographs section that makes it too long to read, too long to be useful and too long to be easilly deleted.
Removes the part stating to add closing tags in the 'why its good for the game' section since they are placed at the top of the PR rather than in the good for the game section.

'Include any screenshots/videos of the modified code functioning successfully ideally including edge cases.' says everything that needs to be said without too much bloat.

New details section (looks the same):
<details>
<summary>Screenshots&Videos</summary>

Put screenshots and Videos documenting testing and execution of intended behaviors here

</details>

## Changelog
:cl:
code: Simplifies the PR template.
/:cl:

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -8,30 +8,15 @@
...
 ## Testing Photographs and Procedure
-<!--
-Include any screenshots, videos, etc. of you testing your code with it successfully functioning.
-Ideally testing should cover:
-Intended use cases(IE: if you are making a shotgun, test it as you intend for it to be used.)
-Potential edge cases(IE: try loading different ammo than you designed for into the shotgun.)
-Please include the steps you went through for the testing(videos are exempt so long as we can see everything being done in frame). Said steps can also help us help you with any issues you encounter during development.
-Pulls from Upstream are generally exempt from this.
--->
+<!-- Include any screenshots/videos of the modified code functioning successfully ideally including edge cases. -->
```
Comment (PowerfulBacon) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
<!-- Include any screenshots/videos/debugging steps of the modified code functioning successfully, ideally including edge cases. -->
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -8,30 +8,15 @@
...
+<!-- Include any screenshots/videos/debugging steps of the modified code functioning successfully, ideally including edge cases. -->
 <details>
-
-
-
 <summary>Screenshots&Videos</summary>
 
-
-
 Put screenshots and Videos documenting testing and execution of intended behaviors here
```
Comment (PowerfulBacon) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.
```

Comment (PowerfulBacon) [reply .github/PULL_REQUEST_TEMPLATE.md]: Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.
